go/tools/builders/cgo.go

```diff
@@ -144,6 +146,16 @@ func run(args []string) error {
 		}
 	}
 
+	if len(cgoSrcs) == 0 {
+		// If there were no cgo sources present, generate a minimal cgo input
+		// This is so we can still run the cgo tool to build all the other outputs
+		nullCgo := filepath.Join(objdir, "_cgo_empty.go")
+		cgoSrcs = append(cgoSrcs, nullCgo)
+		if err := ioutil.WriteFile(nullCgo, []byte("package "+pkg+"\n/* const int ___no_cgo_present; */\nimport \"C\"\n"), 0644); err != nil {
```

Comment:
Will this cause a link error if there is more than one package like this? I think it will work, but it might be better to leave out the comment and just `import "C"`.